src/app/login/login.component.ts

```diff
@@ -42,7 +42,7 @@ export class LoginComponent implements OnInit {
     this.userService.insert(newUser).subscribe(
       (response) => {
         this.resp = response;
-        if (this.resp !== 'User Inserted') {
+        if (this.resp !== '1') {
```

Comment:
response will be a number, not always one

Reply:
this will always be '1' or '0' with new backend setup to avoid token issue